scripts/upgrade

```diff
@@ -37,14 +38,23 @@ if [ -z "$final_path" ]; then
 fi
 
 # Migrate from legacy permission system
-
 is_public=$(ynh_app_setting_get --app=$app --key=is_public)
 
 if [ -n "$is_public" ]; then
-        ynh_app_setting_delete --app=$app --key=skipped_uris
-	if [ $is_public -eq 1 ]; then
+	# Remove unprotected_uris
+	ynh_app_setting_delete --app=$app --key=unprotected_uris
+	# Remove protected_uris
+	ynh_app_setting_delete --app=$app --key=protected_uris
```

Comment:
Considering this app, you can spare `unprotected_uris` and `protected_uris` as the app never used it. So no need to remove it.